Source/ACE/Entity/SpellEntity.cs

```diff
@@ -0,0 +1,58 @@
+﻿using ACE.Entity;
+using ACE.Entity.Enum;
+using ACE.Factories;
+using ACE.Managers;
+using ACE.Network;
+using ACE.Network.Enum;
+using ACE.Network.GameMessages.Messages;
+using log4net;
+
+namespace ACE.Entity
+{
+    /// <summary>
+    /// Spell Entity
+    /// </summary>
+    public class SpellEntity :  MutableWorldObject
```

Comment:
I wanted to keep it separate because I am not sure yet what all it will needed,
But PlayScript may have to be far more advanced as we learn more.

I don't want to put all that logic in all mutable objects, I think it would create a mess as things progress.  for now I thinks its best to keep them separated.

And if we find other objects with different code then PlayScript, we can then make it more generic if we keep them separated.  Code will be a little bigger but much easier to read in the long run as we have different kinds of Entities.